src/react/deckgl.js

```diff
@@ -126,56 +128,55 @@ export default class DeckGL extends React.Component {
       );
     }
 
-    this.events = addEvents(canvas, {
-      cacheSize: false,
-      cachePosition: false,
-      centerOrigin: false,
-      onClick: this._onClick,
-      onMouseMove: this._onMouseMove,
-      onDragStart: this._onDragEvent,
-      onDragMove: this._onDragEvent,
-      onDragEnd: this._onDragEvent,
-      onDragCancel: this._onDragCancel
+    // TODO: add handlers on demand at runtime, not all at once on init
+    this.eventManager = createEventManager(canvas, {})
```

Comment:
Some API design comments:
* Not sure I see a need for the `createEventManager` wrapper. Shouldn't we just export the `EventManager` class?
* Especially since it is a class with methods that the user is expected to call, we'll want to document it as a class next to our other classes.
* Also seems taking the first set of handlers directly in the constructor would be nice - perhaps then the `on`/`off` methods might not be needed for casual users...
```
new `EventManager(canvas, {
   click: this._onClick,
   ...
})
```